ci/docker/core.build.Dockerfile

```diff
@@ -16,3 +16,5 @@ RUN pip install -U --ignore-installed  \
   -r python/requirements.txt \
   -r python/requirements/test-requirements.txt \
   -r python/requirements/ml/dl-cpu-requirements.txt
+
+RUN pip install python/requirements/nsight_fake
```

Comment:
Hmm I feel like we shouldn't include this to actual docker image. Can we only include it to a test images? Or install is separately from the build

Reply:
I'll try move it on basetest.Dockerfile